godot-macros/src/class/godot_api.rs

```diff
@@ -56,6 +56,7 @@ enum BoundAttrType {
         has_gd_self: bool,
     },
     Signal(AttributeValue),
+    #[allow(dead_code)]
     Const(AttributeValue),
```

Comment:
Only the inner `AttributeValue` is unused, there are `Const(_)` match arms. 
Can you apply the attribute to the nested type?